examples/stopwatch.py

```diff
@@ -35,7 +35,18 @@
     Input("stopwatch-x-y", "value"),
 )
 def update_output(n, input_value, start_watch, led_value):
+    """
+    takes in the current value from LED display and reduces it by 1
+    
+    Inputs: 
+        n: number of intervals
+        input_value: number of seconds entered by user
+        start_watch: boolean value to start/stop the stopwatch
+        led_value: current value from LED display
 
+    Output:
+        led_value: current value from LED display
+    """
```

Comment:
Thanks for adding this Somesh, but for consistency purposes with all the other apps, we don't use a function description. I'll delete this file before accepting the PR.

Reply:
I think we do have function docstring for some functions

Reply:
Ok, if you see them, let me know and I'll remove the descriptions. 

Thanks Somesh

Reply:
sure